src/main/java/controller/CarController.java

```diff
@@ -0,0 +1,55 @@
+package controller;
+
+import domain.Car;
+import domain.CarRace;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import utils.Parser;
+import validator.InputValidator;
+import view.GamePrinter;
+import view.GameReader;
+
+public class CarController {
+
+    public void gameRun() {
+        List<Car> carNames = inputCarNames();
+        int gameCount = inputGameCount();
+        gameStart(carNames, gameCount);
+    }
+
+    // 자동차 이름 입력받기
+    private List<Car> inputCarNames() {
+        String carNames = GameReader.getNames();
+        List<String> parsedNames = Parser.parseStringArray(carNames);
+        List<Car> cars = new ArrayList<>();
+        for (String name : parsedNames) {
+            cars.add(new Car(name));
+        }
+        return cars;
```

Comment:
저는 반복성 로직에 대해 최대한 `stream`을 활용하려고 해요.
가독성도 높고, 불필요한 초기화도 필요하지 않아서요!
태현님은 어떤걸 선호하시는 편인가요?

아래처럼 활용할 수 있겠네요!
```suggestion
        List<String> parsedNames = Parser.parseStringArray(carNames);
        return parsedNames.stream()
                .map(Car::new)
                .toList();
```

Reply:
분명 다른 코드들에서는 스트림을 활용했다고 생각했는데 빼먹은 부분이 있었네요! 깔끔하고 좋은 코드 감사합니다